src/services/auth-service.js

```diff
@@ -1,11 +1,33 @@
+const bcrypt = require("bcrypt");
+const config = require("../config/jwtConfig");
+const User = require("../repositories/user");
+const jwt = require("jsonwebtoken");
+
 class AuthService {
-	login(username = "", password = "") {
-		// TODO: Implement login
-		if (username && password) {
-			return `${username}`; // TODO - logic for lgin
-		}
-		return true;
-	}
+  async register(username, password) {
+    const hashedPassword = bcrypt.hashSync(password, 2);
+    await User.save({ username: username, password: hashedPassword });
+  }
+
+  async login(username, password) {
+    const result = await User.get(username);
+    if (result && result.length > 0) {
+      const hashedPassword = result[0].password;
+      const isCorrectPwd = bcrypt.compareSync(password, hashedPassword);
+      if (isCorrectPwd) {
+        return true;
+      } else return false;
```

Comment:
This else block not needed I guess.  Simply `return isCorrectPwd; `

Reply:
Done